ci-bootstrap/terraform/.terraform-version

```diff
@@ -0,0 +1 @@
+0.12.17
```

Comment:
Update to the most recent release of terraform

Reply:
Isn't that 0.12.25 ?

Reply:
Yup I believe so

Reply:
Done